code/datums/religions.dm

```diff
@@ -895,4 +895,18 @@
 	bible_names = list("How to Play Dwarf Fortress", "Book of Grudges", "Strike the Earth", "Lazy Newb Pack", "The Will of Armok", "Mining 101", "Hidden Fun Stuff and You")
 	male_adept = "Expedition Leader"
 	female_adept = "Expedition Leader"
-	keys = list("armok", "dwarf", "dorf", "dwarf fortress", "dorf fort")
\ No newline at end of file
+	keys = list("armok", "dwarf", "dorf", "dwarf fortress", "dorf fort")
+
+/datum/religion/art
+	name = "The Joy of Painting"
+	deity_name = "Bob Ross"
+	bible_names = list("The Joy of Painting")
+	male_adept = "Painter"
+	female_adept = "Painter"
+	keys = list("art", "bob ross", "happy little trees", "happy little clouds")
+
+/datum/religion/art/equip_chaplain(var/mob/living/carbon/human/H)
+	H.put_in_hands(new /obj/item/mounted/frame/painting)
+	H.h_style = "Big Afro"
+	H.f_style = "Full Beard"
```

Comment:
I think female characters don't have sprites for bread and therefore may or may not break with this.

Reply:
They do. It forced the beard regardless of gender, so kept it as a feature